README.md

```diff
@@ -43,6 +43,7 @@ Kubernetes is available at https://192.168.99.100:443.
 
 $ kubectl run hello-minikube --image=gcr.io/google_containers/echoserver:1.4 --hostport=8000 --port=8080
 deployment "hello-minikube" created
+#NOTE: It may take a few minutes for the echoserver pod to be up and running before you can curl it
```

Comment:
What if we change this to include a "kubectl get pods" first, so the user can see something is actually happening. Then we can make a note saying that curl won't work until the state is "Running", which can take a few minutes.